tests/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ endforeach()
 add_subdirectory(psi4numpy)
 add_subdirectory(python)
 add_subdirectory(json)
-add_subdirectory(pasture)
+add_subdirectory(pasture-ccsorttransqt2)
```

Comment:
Looking again, I see what you mean. All the same, for ease of comprehension at a glance, I think seeing a "pasture-ccsorttransqt2/test_name .... PASSED" should mean that ccsorttransqt2 from pasture is installed and working correctly. Similar to the erd test that sets the integrals module to erd, even though it would pass just fine with libint. So I'd remove the logic from the pasture tests and only activate the tests if the transqt2/ccsort field from the pasture is active. Though I'm willing to entertain other views.